openupgrade_scripts/scripts/payment_paypal/16.0.2.0/noupdate_changes.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version='1.0' encoding='utf-8'?>
 <odoo>
   <record id="payment.payment_provider_paypal" model="payment.provider">
-    <field name="code">paypal</field>
-    <field name="redirect_form_view_id" ref="redirect_form"/>
+    <!-- <field name="code">paypal</field>
```

Comment:
Is this a useful comment or the block can be totally removed?

Reply:
It should not be removed as explained in https://github.com/OCA/OpenUpgrade/pull/4178#issuecomment-1772577977.

Reply:
They aren't removed because we want to have a trace of the "diff" if we do a refresh of the analysis (as Odoo changes thing in the middle of the release). Commenting it, next time we refresh the analysis and compare, we see that it's not new noupdate changes, just that we decided not to update them (by whatever reason, that can also be explained as comment).